test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/PackageLicenseUtilitiesTests.cs

```diff
@@ -167,5 +167,24 @@ public void PackageLicenseUtility_GeneratesLinkForFiles()
             Assert.Equal(Resources.Text_ViewLicense, licenseFileText.Text);
             Assert.Equal(Resources.LicenseFile_Loading, ((Run)((Paragraph)licenseFileText.LicenseText.Blocks.AsEnumerable().First()).Inlines.First()).Text);
         }
+
+        [Fact]
+        public void PackageLicenseUtility_GenerateCorrectLink()
+        {
+            // Setup
+            var license = "MIT";
```

Comment:
It should check composite license expression, isn't it? For just `MIT` "wrong" and "right" encoding produce the same results.